app/controllers/api/marketing/lists_controller.rb

```diff
@@ -0,0 +1,665 @@
+# 
```

Comment:
@sapinva also, don't you want to include versioning in the API url here as well? so, this would go in `app/controllers/api/v1/marketing/lists_controller.rb` ?